.circleci/define_build_environment_variables

```diff
@@ -8,23 +8,7 @@ export safe_git_branch=${CIRCLE_BRANCH//\//-}
 export short_sha="$(git rev-parse --short=7 $CIRCLE_SHA1)"
 export IMAGE_TAG="$safe_git_branch.$short_sha"
 
-case $NAMESPACE in
-  production)
-      export NAMESPACE_HOST="${PRODUCTION_HOST}"
-      ;;
-  uat)
-      export NAMESPACE_HOST="${UAT_HOST}"
-      ;;
-  staging)
-      export NAMESPACE_HOST="${STAGING_HOST}"
-      ;;
-  training)
-      export NAMESPACE_HOST="${TRAINING_HOST}"
-      ;;
-  *)
-      return
-      ;;
-esac
+export NAMESPACE_HOST="${HOST}"
```

Comment:
I think line 16 and 19 are the only use of `NAMESPACE_HOST` remaining, what do you think of replacing those with `HOST` and removing `NAMESPACE_HOST` altogether?

Reply:
Oh yeah brain fart moment, amended.